packages/store/ts/config/v2/error.ts

```diff
@@ -0,0 +1 @@
+export type Error<reason extends string = string> = `!Error: ${reason}`;
```

Comment:
would this conflict at all with the native `Error` class/type (e.g. `throw new Error(...)`)?